src/T2DMap.cpp

```diff
@@ -1993,6 +1993,58 @@ void T2DMap::paintAreaExits(QPainter& painter, QPen& pen, QList<int>& exitList,
                     }
                     painter.drawPolyline(polyLinePoints.data(), polyLinePoints.size());
 
+                    int rID = -1;
+                    QMapIterator<int, QString> otherExitIt = room->getOtherMap();
+                    while (otherExitIt.hasNext()) {
+                        otherExitIt.next();
+                        if (otherExitIt.value().startsWith(QLatin1String("0")) || otherExitIt.value().startsWith(QLatin1String("1"))) {
+                            if (otherExitIt.value().mid(1) == itk.key()) {
+                                rID = otherExitIt.key();
+                                break;
+                            }
+                        } else if (otherExitIt.value() == itk.key()) {
+                            rID = otherExitIt.key();
+                            break;
+                        }
+                    }
+                    if (rID > -1) {
+                        TRoom* pE = mpMap->mpRoomDB->getRoom(rID);
```

Comment:
```suggestion
                        auto exitRoom = mpMap->mpRoomDB->getRoom(rID);
```

Reply:
I'd not like to loose track that it is a pointer and not a TRoom instance - my preference would have been:
```diff
-                        TRoom* pE = mpMap->mpRoomDB->getRoom(rID);
+                        auto pExitRoom = mpMap->mpRoomDB->getRoom(rID);
```
🤷‍♂️